src/System.Private.ServiceModel/tests/Scenarios/SelfHostWcfService/WcfService.csproj

```diff
@@ -54,19 +55,51 @@
     <Compile Include="IWcfRestartService.cs" />
     <Compile Include="IWcfService.cs" />
     <Compile Include="MahojongTypes.cs" />
-    <Compile Include="Program.cs" />
```

Comment:
I guess it's not necessary to remove this in case you want to run the WcfService without the bridge. But since I've already changed it to a library instead of a console app I'll just remove it.